app/constants/schedules.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
     THREE_DAY_QUARTERLY: 'T4'
   },
   NAMES: {
-    Q4: 'Quarterly'
+    Q4: 'Quarterly',
+    M12: 'Monthly',
+    T4: 'Quarterly',
+    N0: 'None'
```

Comment:
Needed to support payment schedule calculation.  Also switching default to `N0` = No schedule as previous default was `Q4`.  That means schemes like Vets which have no schedule would default to `Q4` when that is incorrect.